beacon-chain/types/active_state.go

```diff
@@ -197,18 +197,32 @@ func (a *ActiveState) CalculateNewActiveState(block *Block, cState *Crystallized
 	}, newBlockVoteCache), nil
 }
 
+func (a *ActiveState) getBlockHash(block *Block, slot uint64) ([32]byte, error) {
+	sback := block.SlotNumber() - (params.CycleLength * 2)
+
+	if !(sback <= slot && slot < sback+(params.CycleLength*2)) {
+		//need to figure out what to do with this error, what happens then...?
+		return [32]byte{}, fmt.Errorf("couldn't get block hash for slot %v")
```

Comment:
Please provide the second parameter. If this is `slot` then please use `%d` instead of `%v`